guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -4,8 +4,36 @@
 [id="foreman-headline-features"]
 == Headline Features
 
+=== Foreman on Debian 11 (Bullseye)
+
+It is now possible to run Foreman on Debian 11. Users are encouraged to upgrade.
+
 [id="foreman-upgrade-warnings"]
 == Upgrade Warnings
 
+=== `require_ssl_smart_proxies` setting dropped
+
+The `require_ssl_smart_proxies` setting has been dropped and Foreman now behaves as if the value was true (the default). This means it's no longer possible to use reverse DNS instead of client certificates over HTTPS connections. Upgrading users should ensure a valid Public Key Infrastructure (PKI) exists.
```

Comment:
```suggestion
The `require_ssl_smart_proxies` setting has been dropped and Foreman now behaves as if the value was `true` (the default). This means it's no longer possible to use reverse DNS instead of client certificates over HTTPS connections. Upgrading users should ensure a valid Public Key Infrastructure (PKI) exists.
```